docs/docs/llms/transformers/tutorials/fine-tuning/transformers-peft.ipynb

```diff
@@ -1075,7 +1074,7 @@
    "source": [
     "### What's Logged to MLflow?\n",
     "\n",
-    "Let's briefly review what is logged/saved to MLflow as a result of your training. To access the MLflow UI, run `mlflow ui` commands and open https://localhost:PORT (PORT is 5000 by default). Select the experiment \"MLflow PEFT Tutorial\" (or the notebook name when running on Databricks) on the left side. Then click on the latest MLflow Run named `Mistral-7B-SQL-QLoRA-2024-...` to view the Run details.\n",
```

Comment:
seems like docusaurus attempts to parse this link as a real link, i get an error: `Cause: Can't parse URL https://localhost:PORT with base unspecified://`